maintainers/team-list.nix

```
@@ -1091,9 +1091,10 @@ with lib.maintainers;
 
   vim = {
```

We probably should keep it in sync with GitHub team name

```suggestion
  neovim = {
```